src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift

```diff
@@ -156,7 +160,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 	}
 
 	func applicationDidBecomeActive(_ application: UIApplication) {
-		Log.info("Application did become active.", log: .background)
+		Log.info("Application did become active.", log: .appLifecycle)
+
+		if !didSetupUI {
+			setupUI(route)
+			didSetupUI = true
```

Comment:
i think you should also reset `route` after handling it ti avoid it gets handled twice

Reply:
Good point. Done.